README.md

```diff
@@ -77,7 +77,7 @@ We do provide a built, minified version of the SDK, that includes `window.ciscos
 
 In-browser usage is almost the same as Node.js, but it handles the user authentication flow for you. See the [browser guide](https://webex.github.io/spark-js-sdk/guides/browsers/) for more information.
 
-If you're already using a bundler (like [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org/)) you can simply import/require the package and use the above snippet and assign the initialized `team` variable to `window.webexteams`.
+If you're already using a bundler (like [Webpack](https://webpack.js.org/)) you can simply import/require the package and use the above snippet and assign the initialized `team` variable to `window.webexteams`.
```

Comment:
Rather than `webexteams`, we're actually just going to start using `webex`. 

Depending on how many changes this is to your docs, maybe we revisit it. But when I rename the top-level package, `ciscospark` will simply be `webex`.

Reply:
I can change it to webex, it's already webexteams in the readme

Reply:
Okay, we will update the docs holistically later. If there are opportunities to use simply `webex` in the future, let's do so.